packages/core/src/common/menu.ts

```diff
@@ -46,6 +46,16 @@ export class MenuModelRegistry {
         return parent.addNode(actionNode);
     }
 
+    registerSubMenu(subMenuPath: string[], label: string): Disposable {
```

Comment:
I would like to replace `registerSubmenu(menuPath, id, label)` with `registerSubmenu(subMenuPath, label)`, it is more convenient for clients to work with menu paths rather than with ids

Reply:
ok